### DISCUSSION TITLE
How to update doks to the latest version?

### DISCUSSION THREAD

**Discussion by assayire** [thuliteio/doks#681] [Q&A]
Hello

Few weeks back, I started a child theme project, and picking up where I left off. I am trying to update to the latest version of doks (esp. would like to use the automatic menu creation feature, very useful).

I don't find anything in the documentation that says how to update doks. Or did I miss it? Appreciate if you can point to me to the relevant documentation or tell me how to update doks to the latest version.

**Comment by h-enk** [comment]
The Doks child theme uses [Doks as an npm package](https://www.npmjs.com/package/@hyas/doks) (so you can easily update Doks). See the docs: [How to Update](https://getdoks.org/docs/help/how-to-update/) npm packages + see the Release Notes: [how to upgrade](https://github.com/h-enk/doks-child-theme/releases/tag/v0.4.0).

**Comment by ImaCrea** [comment]
Hello, I've followed your instructions but I see in package-lock.json it's still written version 0.3.5 for Doks.

![Capture d’écran 2022-03-08 à 12 28 51](https://user-images.githubusercontent.com/764332/157229849-8eb97b0b-7246-40df-8895-c1b79b3f0f62.png)

Is that ok? What do you recommend? Should I edit that part manually?

Thanks for your precious help @h-enk

**Reply by h-enk** [reply]
In `package.jon` (line 41) replace `"@hyas/doks": "^0.3",` with `"@hyas/doks": "^0.4",`, then run `npm update` — see also [npm-update](https://docs.npmjs.com/cli/v7/commands/npm-update)

Still got problems? Run `npm run clean:install`, then `npm install`
